docs/source/notes/autograd.rst

```diff
@@ -219,24 +219,44 @@ Autograd for Complex Numbers
 **What notion of complex derivative does PyTorch use?**
 *******************************************************
 
-PyTorch follows `JAX's <https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#Complex-numbers-and-differentiation>`_
-convention for autograd for Complex Numbers.
+PyTorch uses the Wirtinger Calculus to compute gradients for functions with complex valued input and (or)
+output. PyTorch's autograd convention for complex numbers allows gradient values to be used in
+Gradient Descent with the same update rule as that in the case of real numbers. This convention works for
+optimization problems where the objective function is real valued.
```

Comment:
It probably would still be helpful to compare with JAX and Tensorflow here (we match TF, and are the opposite of JAX). It also seems like a good idea to tell people (1) that there is a choice AT ALL to be made here, and (2) compare and contrast the benefits of each convention.